### PR TITLE
Add missing documentation for `--no-color` and `--color` options to `sphinx-build` man page

### PR DESCRIPTION
**Pull request by chrysle**
Subject: Add missing documentation for `--no-color` and `--color` options to `sphinx-build` man page

### Feature or Bugfix

- Refactoring

### Purpose
- The `--color` and `--no-color` to `sphinx-build` were added in the linked PR. They are included in the short help message, but lack documentation on `sphinx-build`'s manual page. 

### Relates
- #3397

### CODE REVIEW

File: doc/man/sphinx-build.rst [hunk 1]
```diff
@@ -201,10 +201,20 @@ Options
...
    exclude some references as "known missing".
 
-.. option:: -N
+.. option:: -N, --no-color
 
    Do not emit colored output.
 
+   .. versionchanged:: 1.6
+      Add ``--no-color`` long option.
+
```
Comment (picnixz) [inline doc/man/sphinx-build.rst:210]: I think there is an extra line.

File: doc/man/sphinx-build.rst [hunk 2]
```diff
@@ -201,10 +201,20 @@ Options
...
+
```
Comment (AA-Turner) [inline doc/man/sphinx-build.rst]:
```suggestion
```

File: doc/man/sphinx-build.rst [hunk 3]
```diff
@@ -201,10 +201,19 @@ Options
...
+.. option:: -N, --no-color
 
    Do not emit colored output.
 
+   .. versionchanged:: 1.6
+      Add ``--no-color`` long option.
+
+.. option:: --color
+
+   Do emit colored output. Auto-detected by default.
```
Comment (AA-Turner) [inline doc/man/sphinx-build.rst]:
```suggestion
   Emit colored output. Auto-detected by default.
```

**Comment by chrysle** [comment]
Thank you for reviewing! Updated.

**Review by picnixz — APPROVED** [review]

**Comment by AA-Turner** [comment]
Thanks @chrysle!

A

**Pull request merged**